critiquebrainz/db/users.py

```diff
@@ -0,0 +1,35 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(item):
+    url = "https://gravatar.com/avatar/{0}{1}"
+    email = item['email']
+    if email and item['show_gravatar']:
+        return url.format(md5(email.encode('utf-8')).hexdigest(), "?")
+    else:
+        return url
```

Comment:
@gentlecat why is it hashing id? Also, I have some changes for that function, see my previous message.